docs/rules/terraform_naming_convention.md

```diff
@@ -279,8 +279,8 @@ Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0
 
 ## Why
 
-Naming conventions are optional, so it is not necessary to follow this. 
-But this rule is useful if you want to force the following naming conventions in line with the [Terraform Plugin Naming Best Practices](https://www.terraform.io/docs/extend/best-practices/naming.html).
+Naming conventions are optional, so you don't necessarily have to follow them.
+However, if your team owns Terraform configurations, it can be helpful to have a consistent naming convention.
```

Comment:
The linked documentation doesn't specifically mention naming conventions for label names, so deleted it.